planetary-social/nos
# Add autoscroll feature to flag views

Final part of [#1489](https://github.com/planetary-social/nos/issues/1489) and [#1493](https://github.com/planetary-social/nos/issues/1493)

## Description
This PR adds autoscroll to the new moderation flow if a category is selected from a section and the section's view is hidden. It also fixes a spacing issue caught by @mplorentz, at the top and bottom of the scrollview when the view is scrolled.

## How to test
1. Build the app
2. Click the side menu
3. Click the Settings on the side menu
4. Scroll down to turn on the toggle for new moderation flow
5. Go back to the Feed screen by clicking the feed tab at the bottom of the screen
6. Find an account, Click on the 3 dots at the top right of the account.
7. Select `Flag this user` option. 
8. Select an option from the categories displayed.
9. Select an option on how to send the flag.
10. Select an option on whether to mute a user or not.
11. Click send.
12. Please confirm that:
    - The view scales to a new section if an option is picked from the previous section and the new section's view is not visible.

## Screenshots/Video

https://github.com/user-attachments/assets/c9092949-8c58-4a38-a289-9aa7001a618a
